identity-aware-proxy/IAPClient/IAPClient.cs

```diff
@@ -128,9 +118,8 @@ class IAPClient
 
             // Encryption algorithm must be RSA SHA-256, according to
             // https://developers.google.com/identity/protocols/OAuth2ServiceAccount
-            SecurityKey key = new RsaSecurityKey(
-                Pkcs8.DecodeRsaParameters(privateKey));
-            var signingCredentials = new SigningCredentials(key,
+            var signingCredentials = new SigningCredentials(
```

Comment:
This is awesome!
Since now the sample is pretty straightforward, we can make it frictionless for the customers by exposing an API that gets service account and client Id as input and returns the token as output. Can we implement it in the auth library code as a separate change?

Reply:
That would make things easier for users.  However, it's a lower priority than other stuff I need to do this quarter.  If you want to create a pull request, feel free.